tmail-backend/mailbox/team-mailboxes/src/main/scala/com/linagora/tmail/team/TeamMailbox.scala

```diff
@@ -0,0 +1,26 @@
+package com.linagora.tmail.team
+
+import org.apache.james.core.Username
+import org.apache.james.mailbox.model.{MailboxConstants, MailboxPath}
+
+object Namespace {
+  val TEAM_MAILBOX_NAMESPACE: Namespace = Namespace(MailboxConstants.NAMESPACE_PREFIX_CHAR + "tm")
+}
+
+case class Namespace(value: String) extends AnyVal
+
+case class MailboxName(value: String) extends AnyVal
+
+object TeamMailbox {
+  def from(mailboxPath: MailboxPath): TeamMailbox =
+    TeamMailbox(
+      Namespace(mailboxPath.getNamespace),
+      mailboxPath.getUser,
```

Comment:
Please bear in mind that the user of a team mailbox in the mailbox path is actually its domain...

For a team mailbox `marketting@linagora.com`:

```
Namespace: #TeamMailbox
User: domain.tld
Mailbox name: marketting
````